pkgs/applications/window-managers/picom/picom-allusive.nix

```
@@ -0,0 +1,24 @@
+{ picom, lib, fetchFromGitHub }:
+
+picom.overrideAttrs (oldAttrs: rec {
+  pname = "picom-allusive";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    owner = "allusive-dev";
+    repo = "picom-allusive";
+    rev = version;
+    hash = "sha256-lk4Ll0mi9h3BAqwgOzFQw4WYKnSW9XTl3PjoK2E4WKg=";
+  };
+
+  postInstall = ''
+    chmod +x $out/bin/picom-trans
+  '' + oldAttrs.postInstall;
+
+  meta = with lib; {
+    description = "A fork of picom featuring improved animations and other features";
+    homepage = "https://github.com/allusive-dev/picom-allusive";
+    licenses = with lib.licenses; [ mit mpl20 ];
+    maintainers = with maintainers; oldAttrs.meta.maintainers ++ [ allusive ];
```

You probably shouldn't ask people as maintainers without asking them, 
```suggestion
    maintainers = with maintainers; [ allusive ];
```

---

I was taking that from another pull request i saw since i haven't done this before. Sorry about that.

---

Nw, you didn't know better!